salt/transport/local.py

```diff
@@ -26,7 +27,7 @@ def send(self, load, tries=3, timeout=60):
             #data = json.loads(load)
             #{'path': 'apt-cacher-ng/map.jinja', 'saltenv': 'base', 'cmd': '_serve_file', 'loc': 0}
             #f = open(data['path'])
-            f = open(load['path'])
+            f = salt.utils.fopen(load['path'])
```

Comment:
Since you're at it, how about a `with` here too? It will make sure the handle is close afterwards...